app/graphql/github.js

```diff
@@ -0,0 +1,76 @@
+import Handlebars from 'handlebars';
+import {isIterable} from '../utils/commons';
+
+const TEMPLATES = {
+  tasks:
+`
+{
+  rateLimit {
+    remaining
+  }
+  search(query: "{{query}}", type: ISSUE, first: 100 ) {
```

Comment:
GraphQL has its own implementation of doing variable if you use a client for it

https://github.com/prismagraphql/graphql-request  
https://github.com/alphasights/ember-graphql-adapter

If you use the Ember.js one, you'll have support for models, I think

Reply:
I try to keep the dependency as simple as possible.

Reply:
Probably will use graphql client, when there is a feature that we needed.

Reply:
You have to.

GraphQL variable is special, it's sent separately from the query and has type checking.

You could implement the GraphQL client yourself, but it's better to use battle-tested library, just find the smallest one out there.

https://medium.com/the-graphqlhub/graphql-tour-variables-58c6abd10f56